github/github.go

```diff
@@ -56,6 +56,18 @@ var (
 	AcceptedPermLevels = []string{"write", "admin"}
 )
 
+type UpdateStrategy string
+
+const (
+	// UpdateStrategyDeferToPR the default value for UpdateStrategy
+	UpdateStrategyDeferToPR UpdateStrategy = "deferToPR"
```

Comment:
Could we do`UpdateStrategyLabel` and `"label"` for the names instead of `deferToPR`?